timeless/restaurants/floors/views.py

```diff
@@ -55,12 +53,11 @@ def edit(id):
         )
 
 
-@BP.route("/delete", methods=["POST"])
-@auth.login_required
-def delete():
-    """ Delete floor with id """
-    flash("Delete not yet implemented")
-    return redirect(url_for("floor.list_floors"))
+class Delete(views.DeleteView):
```

Comment:
@emilianodellacasa It seems strange to me that Floors can be deleted... a restaurant's floors are part of the building, they will never be deleted in real life. Maybe deactivated/put on hold/closed temporarily. 

Can we rename Delete to Deactivate? And not remove the Floor, but set a flag on it. WDYT?

Reply:
@amihaiemil you comment is correct, before when we had the same issue with other entities POs suggested that it should be possible to completely destroy data from database. Sometimes, you just made a mistake and deleting is the right fix. We could of course develop the mechanism you suggest, especially adding a flag marking the floor as deleted (and not showing in the list).

@zverovich @virt13  What do you guys think?

Reply:
@emilianodellacasa At the moment, we need only the basic operations above the floors Add / Rename / Delete.
You also need to implement a confirmation when user tries to delete a floor that has at least one table. Something like this: "Do you really want to delete the floor? All tables belonging to this floor will be also deleted!”

Reply:
@zverovich Thanks a lot, noted!